### PR TITLE
Make read_delay in DigitalInputCounter configurable

### PR DESCRIPTION
**Pull request by ba58smith**

### CODE REVIEW

**Comment by mairas** [comment]
Dammit. That was me but on my work account...

**Comment by ba58smith** [comment]
@mairas - Well, at least I can usually figure out how to copy someone else's code. As long as it doesn't require too many changes!

File: src/sensors/digital_input.cpp [hunk 1]
```diff
@@ -61,4 +61,34 @@ void DigitalInputCounter::enable() {
...
+  JsonObject& root = buf.createObject();
+  root["read_delay"] = read_delay;
+  root["value"] = output;
+  return root;
+  };
+
+  static const char SCHEMA[] PROGMEM = R"###({
+    "type": "object",
+    "properties": {
+        "read_delay": { "title": "Read delay", "type": "number" },
```
Comment (joelkoz) [inline src/sensors/digital_input.cpp]: Just FYI - you can add an optional "description" field to the property definition to give a little more information if you'd like.  For example:

``
{ "title": "Read delay", "type": "number", "description": "A delay, in milliseconds, between each read of the input" },
``

Comment (ba58smith) [reply src/sensors/digital_input.cpp]: I will do just that! Thanks.

Comment (ba58smith) [reply src/sensors/digital_input.cpp]: Done. I will merge in a little bit. Taking the boat to the yard for a new coat of bottom paint.

**Pull request merged**